src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx

```diff
@@ -60,20 +58,24 @@ To log into New Relic:
 2. You'll need to create a new password. To do this, click **Forgot your password**. 
 3. Enter your email address. It must be the same one you use for your Azure account. 
 
-## View data in New Relic [#view-newrelic]
+## Next steps in New Relic [#view-newrelic]
 
-Once you're in New Relic, here are some ideas of what to do first: 
+Once you're in New Relic, here are some ideas of what to do next: 
 
 * [View your APM data](/docs/apm/apm-ui-pages/monitoring/apm-summary-page-view-transaction-apdex-usage-data)
-* [View your infrastrucuture data](/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui-entities)
+* [View your infrastructure data](/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui-entities)
 * [View your logs](/docs/logs/ui-data/use-logs-ui)
 * [Set up alerting](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition)
-* [Learn about the New Relic platform](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one)
+* [Add more New Relic integrations](https://newrelic.com/instant-observability).
 
 ## Pricing details [#pricing]
 
 For pricing details and other feature explanations, see [the Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg).
 
+## Integration details [#tech-details]
+
+This service relies on [our Azure Monitor integration](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor).
+
 ## Note about datacenters [#datacenters]
 
 The Azure Native New Relic Service stores your data directly on Microsoft's Azure data centers. Currently the Azure Native service stores all data in Azure's US East region. 
```

Comment:
I'd like our legal team (Joseph Jang) to review the section below about log patterns.